src/panels/KeybindsPanel/KeybindsPanel.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Badge,
+  Code,
+  Container,
+  Divider,
+  Grid,
+  Group,
+  Paper,
+  Table,
+  Text,
+  Title
+} from '@mantine/core';
+
+import { Action } from '@/types/types';
+
+import { FullKeyboard } from './FullKeyboard/FullKeyboard';
+import { KeybindButtons } from './Keybind';
+
+export function KeyBindsPanel() {
+  const [selectedActions, setSelectedActions] = React.useState<Action[]>([]);
+  const [activeModifiers, setActiveModifiers] = React.useState<string[]>([]);
+  const [selectedKey, setSelectedKey] = React.useState<string>('');
+
+  const hasSelectedKeys = selectedKey === '' && activeModifiers.length === 0;
+  return (
+    <Container maw={'none'}>
+      <FullKeyboard
+        setSelectedActions={setSelectedActions}
+        setActiveModifiers={setActiveModifiers}
+        setSelectedKey={setSelectedKey}
+        selectedKey={selectedKey}
+        activeModifiers={activeModifiers}
+      />{' '}
+      <Group my={'md'}>
+        <Title order={3}>Selected keybind:</Title>
```

Comment:
SHould this really be a `Title`? Seems more like a label, IMO

Also, see next comment on title orders